domain/tutoringModel/generate_learning_path.py

```diff
@@ -0,0 +1,109 @@
+from collections import OrderedDict
+import pytest
+
+
+class GenerateLearningPath:
+
+    # Learning Element Types: (+) = 1 and (-) = 0
+    # "KU" and "LK":0 sind für alle Lernenden
+    # gleich relevant / Equally relevant for all learners
+    learning_style_RQ = {"AKT": 0, "REF": 1, "INT": 1}
+    learning_style_SE = {"AKT": 1, "REF": 0, "SNS": 1}
+    learning_style_FO = {"AKT": 1, "REF": 0, "INT": 0, "VIS": 0, "VRB": 1}
+    learning_style_ZL = {"AKT": 0, "REF": 1,
+                         "SNS": 0, "INT": 1, "VIS": 0, "VRB": 1}
+    learning_style_AN = {"AKT": 1, "REF": 0,
+                         "SNS": 1, "INT": 0, "VIS": 1, "VRB": 0}
+    learning_style_ÜB = {"AKT": 1, "REF": 0, "SNS": 1, "INT": 1}
+    learning_style_BE = {"AKT": 0, "REF": 1, "SNS": 1, "INT": 0, "GLO": 1}
+    learning_style_AB = {"SNS": 1, "INT": 0, "GLO": 1}
+    learning_style_ZF = {"REF": 1, "GLO": 1}
+
+    def __init__(self):
+        pass
+
+    def check_learning_style(self, input_learning_style):
+        # input_learning_style= {"AKT": 2, "INT": 7,"VIS": 7, "GLO": 7}
```

Comment:
As many comments as necessary, as few as possible

Reply:
I agree, I ll remove this